lib/Jinja2Filters/strftime.py

```diff
@@ -18,21 +18,25 @@
 from isodatetime.parsers import TimePointParser
 
 
-def strftime(iso8601_datetime, strftime):
+def strftime(iso8601_datetime, strftime, strptime=None):
```

Comment:
I must have missed it last time, but it is best to avoid using the name of the function (and or well-known functions) as argument names.